doc/classes/GeometryInstance.xml

```diff
@@ -14,32 +14,54 @@
 	</methods>
 	<members>
 		<member name="cast_shadow" type="int" setter="set_cast_shadows_setting" getter="get_cast_shadows_setting" enum="GeometryInstance.ShadowCastingSetting">
+			The currently selected shadow casting flag.
 		</member>
 		<member name="extra_cull_margin" type="float" setter="set_extra_cull_margin" getter="get_extra_cull_margin">
+			The extra distance added to the GeometryInstance's bounding box ([AABB]) to increase it's cull box.
 		</member>
 		<member name="lod_max_distance" type="float" setter="set_lod_max_distance" getter="get_lod_max_distance">
+			The max distance for the LOD of this GeometryInstance.
```

Comment:
Per doc guidelines, prefer possessive over "of the" constructions. 

The GeometryInstance's max LOD distance.